cli/src/component/main_view.rs

```diff
@@ -71,8 +72,17 @@ impl Input for MainView {
             state.focus_on(focus::TERMINATION);
             state.update_state();
 
+            // Spawn a new thread to exit the process after 30s if it has not already exited
             if !is_docker_running() {
+                let _ = disable_raw_mode();
                 std::process::exit(0);
+            } else {
+                std::thread::spawn(|| {
```

Comment:
You are not waiting on this task, so will it actually end?

Reply:
I tested it, and it ends when the application exits normally, to it does its thing after 60s